### PR TITLE
Fix #802 - Add notification volume level preference

### LINKED ISSUE #802: Add custom volume for "My reminders" notifications
**Summary:** 

Thanks to @bridgette via https://github.com/OneBusAway/onebusaway-android/pull/801, we now have custom controls over notification sounds and vibration for the "My reminders" feature.

Here's an additional request for notification customization from a user, via Ideascale - http://onebusaway.ideascale.com/a/dtd/Notifications/537373-8715:

>I like the "set alarm" notification, but an additional customization option would be nice so that I can hear the notification. My default tone is not very loud (on purpose), I would set this app to have a louder notification tone - if I could set that.

**Steps to reproduce:** 

1. Tap on arrival time and choose "Set a reminder", and set a reminder to go off for an arrival
1. Listen to the notification volume when it's triggered

Then, go to app "Settings".

**Expected behavior:** 

Give me control over the OBA notification volume independent of system notification volume (most likely in the Settings menu under "Notification" section).

**Observed behavior:** 

There is no way to independently control OBA notification volume

**Device and Android version:** 

N/A

### PR DESCRIPTION
**Pull request by RTsGIT**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `gradlew connectedObaGoogleDebugAndroidTest connectedObaAmazonDebugAndroidTest` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by RTsGIT** [comment]
Fix for issue: https://github.com/OneBusAway/onebusaway-android/issues/802

**Comment by barbeau** [comment]
@RTsGIT Thanks for working on this!  I'll hopefully review in the next few days.

File: onebusaway-android/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -626,6 +626,10 @@
     <string name="preferences_preferred_sound_title">Reminder sound</string>
     <string name="preferences_preferred_sound_summary">Select the OneBusAway notification tone</string>
 
+
+    <string name="preferences_preferred_volume_title">Reminder Volume</string>
+    <string name="preferences_preferred_volume_summary"> Set OneBusAway notification volume</string>
```
Comment (barbeau) [inline onebusaway-android/src/main/res/values/strings.xml]: Extra space at beginning of string should be removed.

File: onebusaway-android/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -626,6 +626,10 @@
     <string name="preferences_preferred_sound_title">Reminder sound</string>
     <string name="preferences_preferred_sound_summary">Select the OneBusAway notification tone</string>
 
+
+    <string name="preferences_preferred_volume_title">Reminder Volume</string>
```
Comment (barbeau) [inline onebusaway-android/src/main/res/values/strings.xml]: Second word should be lower-case - `volume`

**Pull request closed without merging**